monkey/infection_monkey/exploit/smbexec.py

```diff
@@ -159,7 +159,7 @@ def _exploit_host(self):
         except Exception:
             status = ScanStatus.SCANNED
             pass
-        T1035Telem(status, UsageEnum.SMB).send()
+        self.telemetry_messenger.send_telemetry(T1035Telem(status, UsageEnum.SMB))
```

Comment:
Maybe now is the time to check on #1571 ?

Reply:
Probably not now. I wan't to keep focused on the agent refactor project.